### DISCUSSION TITLE
SEOM issue (again) - reporting gone crazy

### DISCUSSION THREAD

**Discussion by zelbanna** [UtilitechAS/amsreader-firmware#881] [General]
Hi,

Something happened again with Kamstrup at SEOM / Sollentuna Energi (new update?). On the 6th of december My readings went bananas

This is my hourly and daily readings:

![image](https://github.com/user-attachments/assets/bb497e0b-5289-4901-8a6e-fbfca58bba03)

I am on 2.3.9, multipliers are all 1,1,1,1

What can I do to troubleshoot?

//Z

**Comment by ArnieO** [comment]
They might have changed the number of decimals on energy (kWh). 
Check what you see in the LCD display on your meter, and compare with what you see in the power gauge:
![image](https://github.com/user-attachments/assets/6bf2498b-9e24-4172-9e76-e3c12d6e323b)

If there is a multiple of ten difference, you need to correct it by using the multipliers.

**Comment by zelbanna** [comment]
You are right, just got changed to 10x :-)
